layouts/shortcodes/observability_pipelines/configure_log_archive/azure_storage/instructions.md

```diff
@@ -0,0 +1,44 @@
+### Create a storage account
+
+Create an [Azure storage account][9051] if you don't already have one.
+
+1. Navigate to [Storage accounts][9052].
+1. Click **Create**.
+1. Select the subscription name and resource name you want to use.
+1. Enter a name for your storage account.
+1. Select a region in the dropdown menu.
+1. Select  **Standard** performance or **Premium** account type. and select the hot or cool access tier.
```

Comment:
```suggestion
1. Select  **Standard** performance or **Premium** account type, and select the hot or cool access tier.
```

Reply:
"hot" and "cool" are capitalized and bolded in the next line, so just want to confirm whether or not they require different formatting here!

Reply:
Oh thanks for catching this! That line about hot and cool storage doesn't belong here. I updated your suggest to remove it.